common/util/src/testFixtures/java/org/eclipse/dataspaceconnector/common/testfixtures/TestUtils.java

```diff
@@ -133,4 +136,17 @@ public static String tempDirectory() {
             throw new RuntimeException(e);
         }
     }
+
+    /**
+     * Create an {@link OkHttpClient.Builder} suitable for using in unit tests. The client configured with long timeouts
+     * suitable for high-contention scenarios in CI.
+     *
+     * @return an {@link OkHttpClient.Builder}.
+     */
+    public static OkHttpClient.Builder testOkHttpClientBuilder() {
```

Comment:
Can we make this return a `OkHttpClient` instead of the `Builder`? In those cases where the builder is actually needed, which are few, we can always call `.newBuilder()`.

Reply:
Thanks for the hint. Will be cleaner indeed.